diff --git a/cli/develop.js b/cli/develop.js
index e6d6831fcb468efac69dd511b273665aa11cfb5e..5020674ebf15dc22dd08e67050f317b0ac9eed54 100644
--- a/cli/develop.js
+++ b/cli/develop.js
@@ -43,4 +43,5 @@ const run = (args) => {
   /* webpack set up */
   const extensionPath = args.extend ? path.resolve(args.extend) : undefined;
   const webpackConfig = generateWebpackConfig({extensionPath, devMode: true});
+  webpackConfig.infrastructureLogging = {level: 'warn'};
   const compiler = webpack(webpackConfig);
@@ -61,4 +62,4 @@ const run = (args) => {
   app.use((webpackDevMiddleware)(
     compiler,
-    {logLevel: 'warn', publicPath: webpackConfig.output.publicPath}
+    {publicPath: webpackConfig.output.publicPath}
   ));
