diff --git a/phyloxml.js b/phyloxml.js
index 61482f5a8beca96e6c7b3f5932dfdf69fb026d49..467bad3086d517150de7494326787685b0467f38 100644
--- a/phyloxml.js
+++ b/phyloxml.js
@@ -90,7 +90,7 @@
     var sax = null;
     if (typeof module !== 'undefined' && module.exports && !global.xmldocAssumeBrowser) {
         // Being used in a Node-like environment
-        sax = require('./sax');
+        sax = require('sax');
     }
     else if (typeof window !== "undefined") {
         // Attached to the Window object in a browser
